docs/docs/how-to/file-uploads.md

```diff
@@ -446,6 +446,50 @@ Now let's add the ability to bring back the uploader if you decide you want to c
 
 We're borrowing the styles from the submit button and making sure that the image has both a top and bottom margin so it doesn't crash into the new button.
 
+## Safe usage of your free plan
+Having a free plan is great, but if you just load images and never actually remove the unnecessary ones, you'll be in trouble.
+
+To avoid this, we'd better implement the `deleteImage` mutation. It will enable you to make a call to the Filestack API to remove your resources, and on success, you will remove the row in the `Image` model.
+
+You are going to need a new `.env` called `REDWOOD_ENV_FILESTACK_SECRET`, which you can find in Filestack > Security > Policy & Signature: App Secret.
+
+Also, let's add this package to the right side of our current concern:
+
+```shell
+yarn workspace api add filestack-js
+```
+
+You should be good to go. Let's modify our service accordingly:
+
+```js {4-23} title="api/src/services/image/image.ts"
+import * as Filestack from 'filestack-js'
+
+export const deleteImage = async({ id }) => {
+  const client = Filestack.init(process.env.REDWOOD_ENV_FILESTACK_API_KEY)
+
+  const image = await db.image.find({ where: { id } })
+
+  /** @manual the `security.handle` is the unique part of the Filestack file's url. */
+  const handle = /\w+$/.exec(image.url).pop()
```

Comment:
I'm assuming this pulls the last part of the URL off? This line is efficient but I feel like it kind of obscures what's going on. I would actually add more code to make it maybe a little more clear what's going on (these docs are usually geared towards a more beginner developer so I don't mind more code that's easier to follow). I didn't even know you could call that `.exec()` function right on a regex like that!

```suggestion
  const handle = image.url.split('/').pop()
```